content/matplotlib/concepts/pyplot/terms/bar/bar.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'bar'
+Description: 'A bar is a chart/graph that represents categorical data with rectangular bars.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A **`bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
+
+## Syntax
+
+```plt.bar(x, height, width, bottom, align)```
```

Comment:
This should be a ```pseudo code block

Reply:
@SSwiniarski could you give an example? If the code is written in a descriptive language, ex: plt.bar(x coordinates, height of the bars, width of the bars, and so on).

 I will then remove the parameters. Would this be okay or am I not getting what you mean by pseudo code?

Reply:
When I say ```pseudo code blocks I am referring to enclosing the content with three backticks followed by the word "pseudo" on its own line, and following with three backtics on its own line. Refer to the [Code](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#code) and ["Supported Code Blocks" ] (https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#supported-code-blocks)part of the style guide.

it should look like this:
```` 
```pseudo
plt.bar(x, height, width, bottom, align)
```
````

Reply:
@SSwiniarski perfect! Thanks for the elaboration.